packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -215,22 +215,37 @@ export function createPlugin(optimizerOptions: OptimizerOptions = {}) {
           // client input default
           opts.input = [path.resolve(srcDir, 'root.tsx')];
         } else if (opts.target === 'lib') {
-          // lib input default
-          opts.input = [path.resolve(srcDir, 'index.ts')];
+          if (typeof updatedOpts.input === 'object') {
+            for (const key in updatedOpts.input) {
+              const resolvedPaths: { [key: string]: string } = {};
+              if (Object.hasOwnProperty.call(updatedOpts.input, key)) {
+                const relativePath = updatedOpts.input[key];
+                const absolutePath = path.resolve(opts.rootDir, relativePath);
+                resolvedPaths[key] = absolutePath;
+              }
+
+              opts.input = { ...opts.input, ...resolvedPaths };
+            }
+          } else {
+            // lib input default
+            opts.input = [path.resolve(srcDir, 'index.ts')];
+          }
         } else {
           opts.input = [];
         }
       }
-      opts.input = opts.input.reduce((inputs, i) => {
-        let input = i;
-        if (!i.startsWith('@') && !i.startsWith('~')) {
-          input = normalizePath(path.resolve(opts.rootDir, i));
-        }
-        if (!inputs.includes(input)) {
-          inputs.push(input);
-        }
-        return inputs;
-      }, [] as string[]);
+      opts.input = Array.isArray(opts.input)
+        ? opts.input.reduce((inputs, i) => {
+            let input = i;
+            if (!i.startsWith('@') && !i.startsWith('~')) {
```

Comment:
can you also include `#` since thats the official alias now

Reply:
Done!😁